docs/quickstart/hello-world-precompile-tutorial.md

```diff
@@ -0,0 +1,942 @@
+      "balance": "0x52B7D2DCC80CD2E4000000"
+    }
+  },
+  "nonce": "0x0",
+  "timestamp": "0x0",
+  "extraData": "0x00",
+  "gasLimit": "0x1312D00",
+  "difficulty": "0x0",
+  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "number": "0x0",
+  "gasUsed": "0x0",
+  "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+}
+```
+
+Adding this to our genesis enables our HelloWorld precompile.
+
+```json
+"helloWorldConfig": {
+  "blockTimestamp": 0
+},
+```
+
+As a reminder, we defined `helloWorldConfig` in `./params/precompile_config.go`. By putting this in genesis, we enable our HelloWorld precompile at blockTimestamp 0.
+
+```go
+// PrecompileUpgrade is a helper struct embedded in UpgradeConfig, representing
+// each of the possible stateful precompile types that can be activated
+// as a network upgrade.
+type PrecompileUpgrade struct {
+	ContractDeployerAllowListConfig *precompile.ContractDeployerAllowListConfig `json:"contractDeployerAllowListConfig,omitempty"` // Config for the contract deployer allow list precompile
+	ContractNativeMinterConfig      *precompile.ContractNativeMinterConfig      `json:"contractNativeMinterConfig,omitempty"`      // Config for the native minter precompile
+	TxAllowListConfig               *precompile.TxAllowListConfig               `json:"txAllowListConfig,omitempty"`               // Config for the tx allow list precompile
+	FeeManagerConfig                *precompile.FeeConfigManagerConfig          `json:"feeManagerConfig,omitempty"`                // Config for the fee manager precompile
+	HelloWorldConfig                *precompile.HelloWorldConfig                `json:"helloWorldConfig,omitempty"`
+	// ADD YOUR PRECOMPILE HERE
+	// {YourPrecompile}Config  *precompile.{YourPrecompile}Config `json:"{yourPrecompile}Config,omitempty"`
+}
+```
+
+Now we can get the network up and running.
+
+Start the server in a terminal in a new tab using avalanche-network-runner. Here's some more information on [avalanche-network-runner](https://docs.avax.network/subnets/network-runner), how to download it, and how to use it.
+
+```bash
+avalanche-network-runner server \
+--log-level debug \
+--port=":8080" \
+--grpc-gateway-port=":8081"
+```
+
+The next steps are to build the Subnet-EVM binary with all the latest precompile changes and the lastest AvalancheGo Binary.
+
+In another terminal tab run this command in the root of the Subnet-EVM repo to get the latest local Subnet-EVM binary. This build script will place the binary in `AVALANCHEGO_PLUGIN_PATH` which we will define later.
+
+```bash
+./scripts/build.sh
+```
+
+Leave the Subnet-EVM repo. Go to the AvalancheGo repo in whatever directory you keep repos and run the command below to get the latest AvalancheGo binary. The following commands build and copy the binary to the `AVALANCHEGO_EXEC_PATH` which we will define later.
```

Comment:
I would suggest that you follow https://docs.avax.network/nodes/build/run-avalanche-node-manually#source-code to specify where to check out the avalanchego project so that the binary is built in the proper location and you can remove 
`cp avalanchego ${HOME}/go/src/github.com/ava-labs/avalanchego/build/avalanchego`
in the step below.

Reply:
Again, for `AVALANCHEGO_EXEC_PATH `, tutorial should be sequential, not back and forth for a reader.

Reply:
I found that I needed to copy the binary to the location to that file path.